src/features/filter.ts

```diff
@@ -1,10 +1,56 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+export enum FilterActionType {
+  setQuery = 'query/SET',
+  setStatus = 'status/SET',
+}
+
+type SetQuery = {
+  type: FilterActionType.setQuery;
+  payload: string;
+};
+
+type SetStatus = {
+  type: FilterActionType.setStatus;
+  payload: Status;
+};
+
+const setQuery = (title: string): SetQuery => ({
+  type: FilterActionType.setQuery,
```

Comment:
Consider renaming the 'title' parameter to 'query' for better clarity.

Reply:
Not fixed